test/unittest/vector_search/test_vector_search.py

```diff
@@ -11,12 +11,15 @@
 
 
 @pytest.fixture
-def index_data():
+def index_data(tmp_path, monkeypatch):
+    unique_dir = tmp_path / f"superduperdb_{uuid.uuid4()}"
```

Comment:
with temp.tempDir ? about using this

Reply:
Done. We also can use `tempfile.TemporaryDirectory` to to this